core/templates/components/forms/schema-based-editors/schema-based-choices-editor.component.spec.ts

```diff
@@ -63,22 +63,22 @@ describe('Schema Based Choices Editor Component', () => {
 
     expect(component.localValue).toEqual(null);
 
-    component.writeValue(true);
+    component.writeValue('true');
     expect(component.localValue).toBeTrue();
```

Comment:
Why is the expectation toBeTrue when the value is a string?

(Also, for clarity, perhaps use test strings that are not "true".)

Reply:
Done.